interfaces/mount/entry.go

```diff
@@ -192,8 +195,63 @@ func OptsToFlags(opts []string) (flags int, err error) {
 		case "strictatime":
 			flags |= syscall.MS_STRICTATIME
 		default:
-			return 0, fmt.Errorf("unsupported mount option: %q", opt)
+			if !strings.HasPrefix(opt, "x-snapd-") {
```

Comment:
Why do we accept x-snapd-* instead of only the well-defined ones?

Reply:
Just future proofing. I can make this an explicit list but not sure if we need that.